frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorHandler.java

```diff
@@ -24,7 +24,7 @@ public boolean submit(
       IdeaLoggingEvent[] events,
       @Nullable String additionalInfo,
       Component parentComponent,
-      Consumer<SubmittedReportInfo> consumer) {
+      Consumer<? super SubmittedReportInfo> consumer) {
```

Comment:
This small change in the header of `com.intellij.openapi.diagnostic.ErrorReportSubmitter#submit` broke **source compatibility** &mdash; but not **binary compatibility**, since the new & old method types are still identical **after erasure**.